notify/notify.go

```diff
@@ -65,3 +71,65 @@ func (n *Notifier) Notify(c context.Context, request *Notification) error {
 		return nil
 	})
 }
+
+func (n *Notifier) addRouter(r *gin.RouterGroup) {
```

Comment:
Just curious why the http stuff (router) was moved from the http package. The notify package ideally was intended to abstract away the http details and be responsible only for sending messages using various internal services.

Reply:
I was trying to follow the breez-lnurl service approach where the router/handler logic is in each package (lnurl & channel packages). But I can refactor